src/main/java/ch/njol/skript/bukkitutil/CommandReloader.java

```diff
@@ -40,11 +41,9 @@
 	static {
 		try {
 			Class<?> craftServer;
-			if (Skript.classExists("org.bukkit.craftbukkit.v1_14_R1.CraftServer")) {
-				craftServer = Class.forName("org.bukkit.craftbukkit.v1_14_R1.CraftServer");
-			} else {
-				craftServer = Class.forName("org.bukkit.craftbukkit.v1_13_R2.CraftServer");
-			}
+			String revision = Bukkit.getServer().getClass().getPackage().getName().split("\\.")[3];
```

Comment:
This might be worth doing outside of the PR, we can release a hotfix release without 1.15-specific features this way.

Reply:
When talking to Bensku a while back, he stated we could do 2.4.1 as a "1.15 update", so I figured since this was such a small change I would knock it all out in one PR.

If you would like, I can remove it.